src/js/components/pipelines/transforms/Inspector.jsx

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var React = require('react'),
+    connect = require('react-redux').connect,
+    capitalize = require('capitalize'),
+    getInVis = require('../../../util/immutable-utils').getInVis,
+    deepEquality = require('../../../util/deep-equality');
+
+function mapStateToProps(state, ownProps) {
+  var id = ownProps.dsId;
+  return {
+    transforms: getInVis(state, 'datasets.' + id + '._transforms')
+  };
+}
+
+var Inspector = React.createClass({
+  propTypes: {
+    dsId:  React.PropTypes.number,
+    transforms:  React.PropTypes.array
+  },
+
+  shouldComponentUpdate: function(nextProps, nextState) {
```

Comment:
Because our store uses [ImmutableJS](https://facebook.github.io/immutable-js/), you don't need this method to test for deep equality in the passed in properties. Instead, ImmutableJS will produce a new ImmutableList for the `transforms` property if it is ever updated, which will have a new reference and thus trigger a re-render.